serverless.yaml

```diff
@@ -163,6 +163,26 @@ functions:
     environment:
       CRAWLER_NAME: !Ref Crawler
 
+  startExportJob:
+    timeout: 30
+    memorySize: 192
+    runtime: nodejs12.x
+    description: 'Start the Glue job for bulk export'
+    role: GlueJobRelatedLambdaRole
+    handler: bulkExport/index.startExportJobHandler
+    environment:
+      GLUE_JOB_NAME: !Ref ExportGlueJob

```

Comment:
safer to use arn instead?

Reply:
why safer?

Job name is sufficient. It is assumed that the job is on the same account and region. Similar to how we only pass ddb table names as env variables.